ios/brave-ios/Sources/AIChat/Components/Messages/AIChatSuggestionsView.swift

```diff
@@ -52,19 +52,19 @@ struct WrappingHStack: Layout {
     }
 
     var x = bounds.minX
-    var y = height / 2.0 + bounds.minY
+    var y = height + bounds.minY
 
     subviews.forEach { subview in
       x += subview.dimensions(in: proposal).width / 2.0
 
       if x + subview.dimensions(in: proposal).width / 2.0 > bounds.maxX {
         x = bounds.minX + subview.dimensions(in: proposal).width / 2.0
-        y += height + vSpacing
+        y += subview.dimensions(in: proposal).height + vSpacing
       }
 
       subview.place(
         at: CGPoint(x: x, y: y),
-        anchor: .center,
+        anchor: .init(x: 0.5, y: 1.0),
```

Comment:
Isn't this the same as `bottom`?

Reply:
Yes correct it is.
I was just being a bit more explicit. But honestly I'll just change it back to .bottom.
It was just easier for me to see that it's 0.5 horizontally (centred), and 1.0 (bottom) :)